### PR TITLE
JWT

### LINKED ISSUE #232: As an engineer, I can initialize plugin-credentials with a JWT token

### LINKED ISSUE #233: As an engineer, I can initialize ciscospark with a JWT token

### PR DESCRIPTION
**Pull request by ianwremmel**
closes #358
Fixes #232
Fixes #233

- [x] Authorize SDK with JWT in nodejs
- [x] Authorize SDK with JWT in browser
- [x] Refresh SDK using JWT in nodejs
- [x] Refresh SDK using JWT in browser
- [ ] Documentation

### CODE REVIEW

**Comment by ianwremmel** [comment]
punting documentation until we're closer to being ready to put something on the portal.

File: packages/node_modules/@ciscospark/plugin-authorization-browser/test/integration/spec/authorization.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ */
+
+import '@ciscospark/plugin-authorization-node';
+import {browserOnly} from '@ciscospark/test-helper-mocha';
+import {assert} from '@ciscospark/test-helper-chai';
+import {createUser} from '@ciscospark/test-helper-appid';
+import CiscoSpark from '@ciscospark/spark-core';
+import uuid from 'uuid';
+
+browserOnly(describe)(`plugin-authorization-node`, () => {
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/plugin-authorization-browser/test/integration/spec/authorization.js]: Should this be `plugin-authorization-browser`?

Comment (ianwremmel) [inline packages/node_modules/@ciscospark/plugin-authorization-browser/test/integration/spec/authorization.js]: yes

**Comment by ianwremmel** [comment]
Needs work: the jwt helper relies on `fetch` which doesn't work in all the browsers we test in.

**Pull request merged**